recipes/clever-toolkit/build.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
-cmake -DCMAKE_INSTALL_PREFIX=$PREFIX -DBOOST_ROOT=$PREFIX -DBoost_NO_SYSTEM_PATHS=ON .
-make
+# TODO remove requirement for ABI=0 once conda-forge has switched to new compiler packages.
```

Comment:
Just for reference, this needs
https://github.com/bioconda/bioconda-utils/issues/284
and 
https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/27
to be done.